prebuilt/boot-assets/config.txt

```diff
@@ -46,6 +46,8 @@ kernel=uboot.bin
 
 dtparam=i2c_arm=on
 dtparam=i2c_vc=on
+#maybe run "sudo modprobe spi-bcm2835" to load kernel modules 
```

Comment:
i dont think that comment is needed, if the kernel does not automatically load the module when the dtparam is set i would consider this a kernel bug ... can you verify if this modprobe call is actually needed ?